commit 06cd09122641b2cc97bd56b8c22bd70905caceff
Author: popcornmix <popcornmix@gmail.com>
Date:   Sat May 26 16:40:51 2012 +0100

    Added power off message to allow kexec to work. Thanks lp0

Index: linux-3.2.46/arch/arm/mach-bcm2708/power.c
===================================================================
--- linux-3.2.46.orig/arch/arm/mach-bcm2708/power.c	2013-07-26 19:33:01.000000000 +0000
+++ linux-3.2.46/arch/arm/mach-bcm2708/power.c	2013-07-26 19:33:08.000000000 +0000
@@ -161,6 +161,7 @@
 	int i;
 
 	printk(KERN_INFO "bcm_power: Broadcom power driver\n");
+	bcm_mailbox_write(MBOX_CHAN_POWER, 0);
 
 	for (i = 0; i < BCM_POWER_MAXCLIENTS; i++)
 		g_state.client_request[i] = BCM_POWER_NOCLIENT;
Index: linux-3.2.46/arch/arm/mach-bcm2708/power.c.rej
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ linux-3.2.46/arch/arm/mach-bcm2708/power.c.rej	2013-07-26 19:33:08.000000000 +0000
@@ -0,0 +1,10 @@
+--- arch/arm/mach-bcm2708/power.c
++++ arch/arm/mach-bcm2708/power.c
+@@ -163,6 +163,7 @@
+    int i;
+ 
+    printk(KERN_INFO "bcm_power: Broadcom power driver\n");
++   bcm_mailbox_write(MBOX_CHAN_POWER, 0);
+ 
+    for (i = 0; i < BCM_POWER_MAXCLIENTS; i++)
+        g_state.client_request[i] = BCM_POWER_NOCLIENT;
